torchvision/transforms/_pil_constants.py

```diff
@@ -0,0 +1,22 @@
+
+import PIL
+from PIL import Image
+
+if tuple(int(part) for part in PIL.__version__.split(".")) >= (9, 1, 0):
```

Comment:
Nit: we don't need the `0` in the version tuple here. Python is checking tuples elementwise. If we hit the case where one tuple has no elements left, the other tuple is greater by definition. For example

```py
>>> (9, 1, 0) >= (9, 1)
True
>>> (9, 1, -1) >= (9, 1)
True
>>> (9, 0, 1) >= (9, 1)
False
>>> (9, 2, 0) >= (9, 1)
True
```

```suggestion
if tuple(int(part) for part in PIL.__version__.split(".")) >= (9, 1):
```